src/main/java/se/sundsvall/paratransit/api/ProcessResource.java

```diff
@@ -35,8 +40,8 @@ class ProcessResource {
 		this.service = service;
 	}
 
-	@PostMapping(path = "/start/{businessKey}")
-	@Operation(description = "Start a new process instance for the provided business key", responses = {
+	@PostMapping(path = "/start/{caseNumber}")
```

Comment:
Missing `produces = APPLICATION_JSON_VALUE` I think? Same goes for the other Post resource